sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdClientChannelPool.java

```diff
@@ -1088,14 +1180,20 @@ private void releaseChannel(final Channel channel, final Promise<Void> promise)
      * {@link #acquire}.
      */
     private void runTasksInPendingAcquisitionQueue() {
+        ensureInEventLoop();
 
         this.ensureInEventLoop();
```

Comment:
Is this same as above ensureInEventLoop?

Reply:
Fixed